src/exo/API_scheduling.py

```diff
@@ -1942,30 +1942,25 @@ def lift_scope(proc, scope_cursor):
 
 
 @sched_op([IfCursorA, BoolA])
-def assert_if(proc, if_cursor, cond):
+def remove_if(proc, if_cursor, cond):
     """
```

Comment:
Why does it take `cond` when it's not passed to `DoRemoveIf` ?

Reply:
Ahhh I forgot to deprecate the API parameter.

Reply:
Done

Reply:
LGTM, I'll delegate it to @skeqiqevian to approve